File: src/main/java/net/softbell/bsh/dto/response/ListResultDto.java
package net.softbell.bsh.dto.response;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : API 수행 결과 리스트 DTO
 */
@Getter
@Setter
public class ListResultDto<T> extends ResultDto
{
    private List<T> list;
}

File: src/main/kotlin/net/softbell/bsh/dto/response/ListResultDto.kt
package net.softbell.bsh.dto.response

/**
 * @author : Bell(bell@softbell.net)
 * @description : API 수행 결과 리스트 DTO
 */
data class ListResultDto<T>(
        var list: MutableList<T> = ArrayList()
) : ResultDto()